docs/enterprise/concepts.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Concepts
+sidebarDepth: 1
+description: Learn how the Pomerium Enterprise Console works.
+---
+
+# Concepts
+
+## Namespaces
+
+In the Pomerium Enterprise Console, a namespace is where you can define an organizational unit of users and groups with fine-grained access management. This enables teams to self-service the routes and policies pertinent to them. Namespaces can optionally inherit from their parent units.
```

Comment:
Should make mention of routes, policies, etc being contained within them as well.  Broadly, they contain resources which you can delegate control over.